Content.Shared/_RMC14/Xenonids/Construction/SharedXenoConstructionSystem.cs

```diff
@@ -199,19 +203,33 @@ private void OnXenoSecreteStructureAction(Entity<XenoConstructionComponent> xeno
         if (attempt.Cancelled)
             return;
 
+        var effectID = XenoStructuresAnimation + choice;
```

Comment:
This should probably be a field on each xeno structure component but for now its fine